.generator/src/generator/templates/api_client.j2

```diff
@@ -181,6 +194,11 @@ module {{ module_name }}
       # set custom cert, if provided
       req_opts[:ssl_ca_file] = File.read(@config.ssl_ca_cert) if @config.ssl_ca_cert
 
+      if @config.debugging
+        @config.logger.debug "HTTP request url ~BEGIN~\n#{url}\n~END~\n"
+        @config.logger.debug "HTTP request header param ~BEGIN~\n#{sanitize_request_header(header_params)}\n~END~\n"
```

Comment:
Can you remove the BEGIN/END bits? I don't think it's useful outside of the body.